mne/beamformer/_rap_music.py

```diff
@@ -31,6 +33,18 @@ def _apply_rap_music(data, info, times, forward, noise_cov, n_dipoles=2,
         Forward operator.
     noise_cov : instance of Covariance
         The noise covariance.
+    nave : int
+        Number of averages.
+    aspect_kind : int
+        The kind of data.
```

Comment:
what value do you expect for aspect_kind?

I think it will always be the same

see what it refers to in :

https://github.com/mne-tools/mne-python/blob/master/mne/io/constants.py#L118